<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XViewerClass</class>
 <widget class="QWidget" name="XViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xviewer.qrc">
    <normaloff>:/XViewer/img/xv.ico</normaloff>:/XViewer/img/xv.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 按钮样式 */
QPushButton:hover

 { 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 102, 184, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(0, 102, 184, 255));
border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 50, 150, 255), stop:0.495 rgba(0, 102, 184, 255), stop:0.505 rgba(0, 102, 184, 255), stop:1 rgba(0, 50, 150, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }



/* 顶部样式 */
#XViewerClass {
    background-color: #333333;
    border-radius: 0px;
}
#head
{
background-color:#3c3c3c;
}
#logo
{
	background-image: url(:/XViewer/img/logo_150_40.png);
}
#close
{
background-image: url(:/XViewer/img/close.png);
}
#min
{
background-image: url(:/XViewer/img/min.png);
}
#max
{
background-image: url(:/XViewer/img/max.png);
}
#normal
{
background-image: url(:/XViewer/img/normal.png);
}
/* body 演示*/
#body
{
background-color: #212121;
}
#left
{
	background-color: #252525;
}
#cams
{
background-color: #1e1e1e;
}


#status
{
	color: rgb(255, 255, 255);
	font: 75 9pt &quot;Adobe Arabic&quot;;
}


/*tab 菜单按钮切换样式 */

#preview
{
background-color: rgb(50, 50, 50);
}
#playback
{
background-color: rgb(50, 50, 50);
}

#preview:checked
{
background-color: rgb(0, 102, 184);
}
#playback:checked
{
background-color: rgb(0, 102, 184);
}</string>
  </property>
  <widget class="QWidget" name="head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="head_button" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="min">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="max">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>normal</zorder>
    <zorder>close</zorder>
    <zorder>min</zorder>
    <zorder>max</zorder>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>监控中。。。</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preview">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>32</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playback">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>32</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>回放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="body" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>801</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QWidget" name="left" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QListWidget" name="cam_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>39</y>
       <width>201</width>
       <height>961</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>camear_test</string>
      </property>
      <property name="icon">
       <iconset resource="xviewer.qrc">
        <normaloff>:/XViewer/img/cam.png</normaloff>:/XViewer/img/cam.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="add_cam">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_cam">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>5</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_cam">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cams" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>501</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="playback_wid" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="XCalendar" name="cal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>431</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="time_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>800</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>20:30:59</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XCalendar</class>
   <extends>QCalendarWidget</extends>
   <header>xcalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>722</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>MaxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normal</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>NormalWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>AddCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>DelCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preview</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>-44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playback</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Playback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>-72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cam_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectCamera(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal</sender>
   <signal>clicked(QDate)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>-94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time_list</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>PlayVideo(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>-39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MaxWindow()</slot>
  <slot>NormalWindow()</slot>
  <slot>AddCam()</slot>
  <slot>SetCam()</slot>
  <slot>DelCam()</slot>
  <slot>Preview()</slot>
  <slot>Playback()</slot>
  <slot>SelectCamera(QModelIndex)</slot>
  <slot>SelectDate(QDate)</slot>
  <slot>PlayVideo(QModelIndex)</slot>
 </slots>
</ui>
